src/handlers/monitorCloudFormationStack.test.ts

```diff
@@ -25,25 +22,18 @@ describe("monitorCloudFormationStack:generateDeleteParams", () => {
         responseElements: null
       }
     };
-    expect(generateDeleteParams(event)).toStrictEqual({
-      Key: {
-        stackName: {
-          S: "lendi-datadog-log-archive-management"
-        },
-        stackId: {
-          S:
-            "arn:aws:cloudformation:ap-southeast-2:12345:stack/lendi-datadog-log-archive-management/16921510-a9e8-11e9-a24e-02d286d7265a"
-        }
-      },
-      TableName: config.DEFAULT_DYNAMODB_TABLE
+    expect(generateDeleteItem(event)).toStrictEqual({
+      stackName: "lendi-datadog-log-archive-management",
+      stackId:
+        "arn:aws:cloudformation:ap-southeast-2:12345:stack/lendi-datadog-log-archive-management/16921510-a9e8-11e9-a24e-02d286d7265a"
```

Comment:
Could we take out any company related info from here? 

- [ ] remove Lendi reference
- [ ] remove datadog reference

Reply:
done